obspy/imaging/waveform.py

```diff
@@ -712,12 +721,20 @@ def __plotStraight(self, trace, ax, *args, **kwargs):  # @UnusedVariable
             # set starttime and calculate endtime
             trace.stats.starttime = self.starttime
         trace.data = np.require(trace.data, np.float64) * calib
+        if self.type == 'relative':
+            start = 0
```

Comment:
Myup.. actually only just now noticed after fixing the PR in that regard.. ;-)
Gonna push the fix in a couple of minutes..